### PR TITLE
buttons and inputs inside graph

### PR DESCRIPTION
**Pull request by dqnykamp**
Addresses part of #1915

### CODE REVIEW

**Comment by dqnykamp** [comment]
PR is last 3 commits.

File: src/Core/components/TriggerSet.js [hunk 1]
```diff
@@ -124,4 +147,58 @@ export default class triggerSet extends InlineComponent {
       this.coreFunctions.resolveAction({ actionId });
     }
   }
+
+
+  async moveButton({ x, y, z, transient, actionId,
```
Comment (jaltekruse) [inline src/Core/components/TriggerSet.js:152]: If there is a reason that it is hard to share this code with the identical copy in UpdateValue, do you want to at least add a comments above each copy pointing to each other, to remind people to keep them in sync?

**Review by jaltekruse — APPROVED** [review]
+1

File: src/Core/components/CallAction.js [hunk 2]
```diff
@@ -209,4 +234,58 @@ export default class CallAction extends InlineComponent {
       this.coreFunctions.resolveAction({ actionId });
     }
   }
+
+
+  async moveButton({ x, y, z, transient, actionId,
```
[the remark this answers is not in the document]
Comment (dqnykamp) [reply src/Core/components/CallAction.js:239]: I found that identical code in 12 places and moved it to a utility function.

**Pull request merged**